resources/views/home.blade.php

```diff
@@ -1,50 +1,182 @@
+                </div>
+            </div>
+            <div class="absolute top-0 mt-20 w-full sm:static sm:w-1/2 sm:mt-0 lg:mb-16">
+                <svg class="bubbles max-w-full h-auto mx-auto mx-auto w-64 sm:w-auto" style="max-height: 516px; padding-bottom: 6%;" viewBox="0 0 430 323"  fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <g class="b1"><path fill-rule="evenodd" clip-rule="evenodd" d="M362.441 103.013H221.62c-9.132 0-17.891 3.673-24.349 10.209-6.458 6.537-10.086 15.403-10.086 24.647v46.596c0 9.245 3.628 18.11 10.086 24.647 6.458 6.537 15.217 10.209 24.349 10.209h140.821c8.651.004 16.991-3.268 23.38-9.173l.362.367C397.42 221.889 415 219.321 415 219.321c-20.48-23.115-17.761-48.248-17.761-48.248l-.181-33.204c-.067-9.255-3.742-18.109-10.224-24.637-6.483-6.527-15.249-10.2-24.393-10.219z" fill="#047A9F" opacity=".9" filter="url(#filter0_dd)"/></g>
+                    <g class="b2"><path fill-rule="evenodd" clip-rule="evenodd" d="M314.759 181.159H173.938c-9.132 0-17.891 3.672-24.349 10.209-6.458 6.537-10.086 15.402-10.086 24.647v46.596c0 9.244 3.628 18.11 10.086 24.647 6.458 6.536 15.217 10.209 24.349 10.209h140.821c8.651.003 16.991-3.269 23.38-9.173l.362.367c11.237 11.374 28.817 8.806 28.817 8.806-20.48-23.115-17.761-48.248-17.761-48.248l-.182-33.204c-.066-9.256-3.741-18.11-10.223-24.637-6.483-6.527-15.249-10.2-24.393-10.219z" fill="#F9587C" opacity=".6" filter="url(#filter1_dd)"/></g>
+                    <g class="b3"><path fill-rule="evenodd" clip-rule="evenodd" d="M76.42 63.278h164.564a40.363 40.363 0 0 1 28.455 11.717 39.884 39.884 0 0 1 11.786 28.288v53.48a39.886 39.886 0 0 1-11.786 28.288 40.364 40.364 0 0 1-28.455 11.717H76.42A40.629 40.629 0 0 1 49.1 186.24l-.424.421C35.545 199.715 15 196.768 15 196.768c23.933-26.53 20.756-55.375 20.756-55.375l.212-38.11a40.183 40.183 0 0 1 11.947-28.276A40.66 40.66 0 0 1 76.42 63.278z" fill="#58B9E9" opacity=".7" filter="url(#filter2_dd)"/></g>
+                    <g class="b4"><path fill-rule="evenodd" clip-rule="evenodd" d="M109.945 148.046h119.534a29.186 29.186 0 0 1 20.668 8.585 29.351 29.351 0 0 1 8.562 20.726v39.183c0 7.774-3.08 15.229-8.562 20.726a29.186 29.186 0 0 1-20.668 8.585H109.945a29.37 29.37 0 0 1-19.846-7.714l-.307.309c-9.538 9.564-24.46 7.405-24.46 7.405 17.383-19.438 15.075-40.572 15.075-40.572l.154-27.922a29.566 29.566 0 0 1 8.678-20.717 29.403 29.403 0 0 1 20.706-8.594z" fill="#EE215A" opacity=".9" filter="url(#filter3_dd)"/></g>
+                    <g class="b5"><path fill-rule="evenodd" clip-rule="evenodd" d="M206.634 5h119.534a29.187 29.187 0 0 1 20.668 8.585 29.35 29.35 0 0 1 8.561 20.725v39.184a29.35 29.35 0 0 1-8.561 20.725 29.187 29.187 0 0 1-20.668 8.585H206.634a29.37 29.37 0 0 1-19.846-7.713l-.307.309c-9.538 9.564-24.461 7.404-24.461 7.404 17.384-19.437 15.076-40.572 15.076-40.572l.154-27.922a29.57 29.57 0 0 1 8.678-20.717A29.405 29.405 0 0 1 206.634 5z" fill="#BB0646" opacity=".8" filter="url(#filter4_dd)"/></g>
+                    <defs>
```

Comment:
Thoughts on pulling this SVG into a partial?

Reply:
I like it - done.